### DISCUSSION TITLE
Is Staticman actively maintained?

### DISCUSSION THREAD

**Discussion by chesterbr** [eduardoboucas/staticman#464] [Q&A] [answered]
Hi! I'm wondering if Staticman is actually maintained (and, if not, whether there is a fork that is).

Asking that because it's a great project (which I am using on [my blog](https://chester.me/) - even [wrote](https://chester.me/archives/2024/02/migrating-blog-comments-from-disqus-to-staticman/) about it), and I'd like to work on something¹ myself, but the last commit is from 4 years ago; there are several issues and seemingly important PRs (such as #450) open.

@eduardoboucas please let us know whether issues and PRs are welcome - totally understand if it's not a priority for you to maintain the project, we just need to know to figure out the best way moving forward. In any case, thank you for putting the work and bridging this gap in such an elegant way! 🔥

(¹ a migration to Nodemailer or some other generic mail sending library because Mailgun has no free tier anymore and a free or cheap alternative is needed; if anything like that already exists, please let me know 🙇‍♂️)

**Comment by alexwaibel** [comment, marked as answer]
Thanks for the kind words about the project. We've been searching for someone to take this over for some time, see #428. Unfortunately there haven't been any volunteers and neither myself or Eduardo have been active on this project in a while. There's a few suggested alternatives in #452 but I haven't tried any of them myself. The cloudflare worker based alternative in that thread seems promising to me and is probably how I'd go about deploying something like Staticman now that Heroku free tier is gone.

**Comment by chesterbr** [comment]
@alexwaibel Thank you so much for the detailed answer! I will look at the alternatives you suggested. Right now I don't feel capable of volunteering, but if I continue using Staticman and feel more at ease with the stack in the future I will let you know!
